packages/utils/src/bulk-query-patients.ts

```diff
@@ -63,14 +65,11 @@ async function queryDocsForPatient(patient: PatientDTO) {
     const queryStartTime = Date.now();
     while (Date.now() - queryStartTime < maxQueryDurationMs) {
       const docQueryStatus = await metriportAPI.getDocumentQueryStatus(patient.id);
-      // ensure at least 1 doc was downloaded, and both the download + conversion have completed
+      // ensure at least 1 doc was returned
       if (
         docQueryStatus.download &&
-        docQueryStatus.download.status === "completed" &&
-        docQueryStatus.download.successful &&
-        docQueryStatus.download.successful > 0 &&
-        (!docQueryStatus.convert ||
-          (docQueryStatus.convert && docQueryStatus.convert.status === "completed"))
+        docQueryStatus.download.total &&
+        docQueryStatus.download.total > 0
```

Comment:
What's the reason for changing this logic?

Reply:
Intention here is to kick off queries, takes too many resources to track completion